src/Bielu.Examine.Umbraco.Forms/Configuration/ConfigureUmbracoFormsIndexOptions.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Examine.Lucene.Analyzers;
+using Examine.Lucene;
+using Examine;
+using Lucene.Net.Analysis;
+using Microsoft.Extensions.Options;
+using Umbraco.Cms.Infrastructure.Examine;
+using Umbraco.Forms.Examine.Indexes;
+
+namespace Bielu.Examine.ElasticSearch.Umbraco.Form.Configuration;
+public class ConfigureUmbracoFormsIndexOptions :
+    IConfigureNamedOptions<LuceneDirectoryIndexOptions>,
+    IConfigureOptions<LuceneDirectoryIndexOptions>
```

Comment:
I lately learned that it is better use IPostConfigureOptions than IConfigureOptions if we override values in default configuration. 

But I am not 100% sure why we need modify it, I am 99% sure that umbraco forms register this anyway?

Reply:
The default ConfigureUmbracoFormsIndexOptions of Umbraco Forms uses 

`options.FieldDefinitions = (FieldDefinitionCollection) new UmbracoFieldDefinitionCollection();`

This adds unnecessary fields which are not related to forms. Please check this in the dashboard.